src/var.cpp

```diff
@@ -410,6 +410,7 @@ void jitc_value_print(const Variable *v, bool graphviz = false) {
         break;
 
     switch ((VarType) v->type) {
+        case VarType::Float16: JIT_LITERAL_PRINT(uint16_t, unsigned, "%u");
```

Comment:
`jitc_value_print` is meant for human-readable outputs. I think it would make sense if printed the floating point value rather than its `uint16_t` representation.

Reply:
I see now that you're doing this in `jitc_var_str()`. I think we would something similar here.